src/components/Button/index.jsx

```diff
@@ -2,15 +2,21 @@ import React from "react";
 import { Link } from "react-router-dom";
 import styles from "./styles/styles.module.scss";
 
-const Button = ({ path, type, children }) => {
+const Button = ({ path, type, children, isFormValid }) => {
```

Comment:
Button is a generic component. So:
1. It should not care about the `isFormValid`. Instead, this logic should be implemented inside the place that this button is used. In your case, I guess ButtonBox. Instead this component can receive `disabled` prop.
2. Handeling `path` is not responsibility of a generic component as well. So move that condition to the upper component. This component should only care about the incoming `path` to render.

Reply:
I'm not sure that I could handle the first one or not
but thanks a lot for your descriptions